--- a/script.js
+++ b/script.js
@@ -61,7 +61,7 @@
   followButtonStyle: 'monochrome',
   hideAdsNav: true,
   hideBookmarkButton: false,
-  hideBookmarkMetrics: true,
+  hideBookmarkMetrics: false,
   hideBookmarksNav: false,
   hideChatNav: false,
   hideCommunitiesNav: false,
@@ -70,7 +70,7 @@
   hideCreatorStudioNav: true,
   hideExplorePageContents: true,
   hideFollowingMetrics: true,
-  hideForYouTimeline: true,
+  hideForYouTimeline: false,
   hideGrokNav: true,
   hideGrokTweets: false,
   hideInlinePrompts: true,
@@ -97,7 +97,7 @@
   hideTwitterBlueUpsells: true,
   hideUnavailableQuoteTweets: true,
   hideVerifiedNotificationsTab: true,
-  hideViews: true,
+  hideViews: false,
   hideWhoToFollowEtc: true,
   listRetweets: 'ignore',
   mutableQuoteTweets: true,
@@ -123,7 +123,7 @@
   tweakNewLayout: false,
   tweakQuoteTweetsPage: true,
   twitterBlueChecks: 'replace',
-  unblurSensitiveContent: false,
+  unblurSensitiveContent: true,
   uninvertFollowButtons: true,
   // Experiments
   customCss: '',
@@ -7517,4 +7517,4 @@
 main()
 //#endregion
 
-}()
\ No newline at end of file
+}()
